fastapi/dependencies/utils.py

```diff
@@ -100,22 +100,19 @@ def check_file_field(field: ModelField) -> None:
                 from multipart.multipart import parse_options_header  # type: ignore
 
                 assert parse_options_header
-            except ImportError:
+            except ImportError as e:
                 logger.error(multipart_incorrect_install_error)
-                raise RuntimeError(multipart_incorrect_install_error)
```

Comment:
This is not refactoring but behavioural change. It will no longer show "while handling the Exception, another exception happened" but "this is cause by". I am not sure if it is clearer.

Reply:
On second thought, I think I tend to oppose this change. The stack trace of the 'old' way mentions `During handling of the above exception, another exception occurred:` which is exactly what is happening. Meaning, `the above exception` refers to the ImportError, while the `another exception` refers to the RuntimeError. 

In your suggestion, this text will change to  `The above exception was the direct cause of the following exception:` where as this is not correct. The `above exception` is not a result from raising a RuntimeError. Hence, I think this might be confusing for users of FastAPI.